docs/reference/migration/migrate_8_0/ilm.asciidoc

```diff
@@ -0,0 +1,17 @@
+[float]
+[[breaking_80_ilm_changes]]
+=== Index Lifecycle Management changes
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+
+// end::notable-breaking-changes[]
```

Comment:
I don't think this is needed?

Reply:
That header was added to all of the subsection documents in https://github.com/elastic/elasticsearch/pull/40990, I just added it to be in line with the rest of the documents in this directory. 

I can remove it if it's not necessary but @lcawl may want to weigh in as she made the original changes.

Reply:
Got this message from @lcawl, GitHub is giving her trouble posting comments but she said this:
> If you want to have the option to tag content in this page and have it appear in https://www.elastic.co/guide/en/elastic-stack/master/elasticsearch-breaking-changes.html too, please do leave that empty tagged section (or put it around your poll interval section)

The poll interval thing definitely does not deserve to be marked as notable - I would be shocked if any real cluster had it set to less than 1 second - but we may have notable breaking changes to ILM sometime before 8.0 so I'm going to leave the empty tagged section there.